### PR TITLE
Move the openshift folder to the gitlab main config repo

### PR DESCRIPTION
**Pull request by krasi-georgiev**
merge only after we migrate and test the other config repo.

Signed-off-by: Krasi Georgiev <8903888+krasi-georgiev@users.noreply.github.com>

### CODE REVIEW

File: build_deploy.sh [hunk 1]
```diff
@@ -1,4 +0,0 @@
-#!/bin/bash
-# This is a placeholder file needed by the CI/CD templates
-# You can safely ignore this file :)
-exit 0
```
Comment (krasi-georgiev) [inline build_deploy.sh:4]: Is this file safe to remove?

**Comment by brancz** [comment]
I think for us to be able to do the migration I would like to merge this piece as the very last thing, only once we've merged and migrated the replacement pieces.

**Comment by krasi-georgiev** [comment]
> I think for us to be able to do the migration I would like to merge this piece as the very last thing, only once we've merged and migrated the replacement pieces.

yes absolutely! 

Waiting for reviews in https://gitlab.cee.redhat.com/observatorium/configuration/merge_requests/29

**Comment by krasi-georgiev** [comment]
will merge after the staging works as expected.

**Pull request merged**